content/docs/command-reference/get-url.md

```diff
@@ -33,14 +33,14 @@ directory will be placed inside.
 
 DVC supports several types of (local or) remote locations (protocols):
 
-| Type    | Description    | `url` format                               |
-| ------- | -------------- | ------------------------------------------ |
-| `local` | Local path     | `/path/to/local/data`                      |
-| `s3`    | Amazon S3      | `s3://mybucket/data`                       |
-| `gs`    | Google Storage | `gs://mybucket/data`                       |
-| `ssh`   | SSH server     | `ssh://user@example.com:/path/to/data`     |
-| `hdfs`  | HDFS to file\* | `hdfs://user@example.com/path/to/data.csv` |
-| `http`  | HTTP to file\* | `https://example.com/path/to/data.csv`     |
+| Type    | Description    | `url` format                                   |
+| ------- | -------------- | ---------------------------------------------- |
+| `local` | Local path     | `/path/to/local/data`                          |
+| `s3`    | Amazon S3      | `s3://mybucket/data`                           |
+| `gs`    | Google Storage | `gs://mybucket/data`                           |
+| `ssh`   | SSH server     | `ssh://user@example.com/abs/path/to/data`      |
+| `hdfs`  | HDFS to file\* | `hdfs://user@example.com/abs/path/to/data.csv` |
```

Comment:
why do we change HDFS? was there a confusion about it?

Reply:
I think so. See https://github.com/iterative/dvc/issues/4167#issuecomment-656468558

Reply:
that comment applies to SSH only, I have no idea about HDFS. With SSH the problem that commands like SCP resolve path in a certain way when you specify it, and you can pass a relative one, etc. So, there was error and trial process to get the path right.

Reply:
OK. I rolled back the "absolute" mentions in HDFS URLs.